upgrading/migrating_etcd.adoc

```diff
@@ -118,38 +118,67 @@ endif::[]
 The following procedure describes the steps required to successfully migrate the
 cluster (implemented as part of the Ansible etcd migration playbook).
 
-. +++<b>Create an etcd backup.</b>+++ See
+. Create an etcd backup. See
 xref:../admin_guide/backup_restore.adoc#cluster-backup[Backup and Restore]
```

Comment:
Please link to the day 2 guide instead. I think the link should be xref:../day_two_guide/host_level_tasks.adoc#day-two-guide-etcd-backup[etcd backup].